src/ripple/nodestore/Database.h

```diff
@@ -209,11 +209,24 @@ class Database : public Stoppable
     void
     onStop();
 
+    /** @return The earliest ledger sequence
+    */
+    static
+    std::uint32_t
+    genesisSeq()
+    {
+        return genesisSeq_;
+    }
+
 protected:
     beast::Journal j_;
     Scheduler& scheduler_;
     int fdLimit_ {0};
 
+    // The earliest ledger sequence. Default is 32570
+    // May be set otherwise for tests or alternate network
+    static std::uint32_t genesisSeq_;
```

Comment:
Is there any thread safety issues? I don't see any and suspect you've already thought about it, but want to raise anyway.

Reply:
No, but I'll make the member private for additional safety.